ppml/trusted-bigdata/Dockerfile

```diff
@@ -0,0 +1,218 @@
+    rm /opt/flink-${FLINK_VERSION}/lib/log4j-core-2.12.1.jar && \
+    rm /opt/flink-${FLINK_VERSION}/lib/log4j-slf4j-impl-2.12.1.jar && \
+    wget -P /opt/flink-${FLINK_VERSION}/lib/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-1.2-api/2.17.1/log4j-1.2-api-2.17.1.jar && \
+    wget -P /opt/flink-${FLINK_VERSION}/lib/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-api/2.17.1/log4j-api-2.17.1.jar && \
+    wget -P /opt/flink-${FLINK_VERSION}/lib/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.17.1/log4j-core-2.17.1.jar && \
+    wget -P /opt/flink-${FLINK_VERSION}/lib/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-slf4j-impl/2.17.1/log4j-slf4j-impl-2.17.1.jar
+RUN ls -al /opt
+
+# stage.2 gramine
+FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG
+
+ARG SPARK_VERSION
+ARG TINI_VERSION
+ARG FLINK_VERSION=1.11.3
+ARG SCALA_VERSION=2.12
+ARG SCALA_SUB_VERSION_NUM=10
+
+ENV SPARK_VERSION                       ${SPARK_VERSION}
+ENV SPARK_HOME                          /ppml/work/spark-${SPARK_VERSION}
+ENV JAVA_HOME                           /opt/jdk8
+ENV PATH                                ${JAVA_HOME}/bin:${PATH}
+ENV SCALA_VERSION                       ${SCALA_VERSION}
+ENV SCALA_SUB_VERSION_NUM               ${SCALA_SUB_VERSION_NUM}
+ENV LOCAL_IP                            127.0.0.1
+ENV SGX_MEM_SIZE                        32G
+ENV SGX_LOG_LEVEL                       error
+ENV SPARK_MASTER_IP                     127.0.0.1
+ENV SPARK_MASTER_PORT                   7077
+ENV SPARK_MASTER_WEBUI_PORT             8080
+ENV SPARK_MASTER                        spark://$SPARK_MASTER_IP:$SPARK_MASTER_PORT
+ENV SPARK_WORKER_PORT                   8082
+ENV SPARK_WORKER_WEBUI_PORT             8081
+ENV SPARK_DRIVER_PORT                   10027
+ENV SPARK_DRIVER_BLOCK_MANAGER_PORT     10026
+ENV SPARK_DRIVER_IP                     $LOCAL_IP
+ENV SPARK_BLOCK_MANAGER_PORT            10025
+ENV TINI_VERSION                        $TINI_VERSION
+ENV LC_ALL                              C.UTF-8
+ENV LANG                                C.UTF-8
+ENV FLINK_VERSION			            ${FLINK_VERSION}
+ENV FLINK_HOME				            /ppml/work/flink-${FLINK_VERSION}
+ENV FLINK_JOB_MANAGER_IP		        127.0.0.1
+ENV FLINK_JOB_MANAGER_REST_PORT		    8081
+ENV FLINK_JOB_MANAGER_RPC_PORT		    6123
+ENV FLINK_TASK_MANAGER_IP		        127.0.0.1
+ENV FLINK_TASK_MANAGER_DATA_PORT	    6126
+ENV FLINK_BLOB_SERVER_PORT              6124
+ENV FLINK_TASK_MANAGER_RPC_PORT		    6125
+ENV FLINK_TASK_MANAGER_TASKSLOTS_NUM	1
+
+RUN mkdir -p /ppml/work && \
+    mkdir -p /ppml/work/lib && \
+    mkdir -p /ppml/work/keys && \
+    mkdir -p /ppml/work/password && \
+    mkdir -p /ppml/work/data && \
+    mkdir -p /ppml/work/models && \
+    mkdir -p /ppml/work/apps && \
+    mkdir -p /root/.keras/datasets && \
+    mkdir -p /root/.zinc && \
+    mkdir -p /root/.m2 && \
+    mkdir -p /root/.kube/
+
+COPY --from=bigdata /opt/jdk  /opt/jdk8
```

Comment:
jdk is not needed after xiangyu's pr

Reply:
jdk will be delete after xiangyu'pr merge